devicetypes/smartthings/rgbw-light.src/rgbw-light.groovy

```diff
@@ -257,7 +261,9 @@ def setColorTemperature(temp) {
 	log.debug "setColorTemperature($temp)"
 	def warmValue = ((COLOR_TEMP_MAX - temp) / COLOR_TEMP_DIFF * 255) as Integer
 	def coldValue = 255 - warmValue
-	def cmds = [zwave.switchColorV3.switchColorSet(red: 0, green: 0, blue: 0, warmWhite: warmValue, coldWhite: coldValue)]
+	def cmds = [zwave.switchColorV3.switchColorSet(red: 0, green: 0, blue: 0, warmWhite: warmValue, coldWhite: coldValue),
+				zwave.basicV1.basicSet(value: 0xFF),
```

Comment:
I think making these conditional was a good idea.

Reply:
Oh, right! Sorry, I don't know why I missed it.